ccd-definition/ComplexTypes/CourtLocation.json

```diff
@@ -3,15 +3,17 @@
     "ID": "CourtLocation",
     "ListElementCode": "applicantPreferredCourt",
     "FieldType": "Text",
```

Comment:
thoughts on this being FieldType: Number?

Reply:
Is text with the regex better and more future proofed than number?

Reply:
It is currently text with regex @AlisonRevittHM

Reply:
oh, it seems I wrote Text to Number by mistake on the commit message - I meant from Number to Text!

Reply:
no worries. glad we're of the same viewpoint. cheers

Reply:
So what are we saying here in the end.. Should it be changed to Number?

Reply:
@ckalista there were implications using interpolation with a number field. So text with regex